assets/xml/objects/object_obj_yasi.xml

```diff
@@ -1,11 +1,11 @@
 ﻿<Root>
     <File Name="object_obj_yasi" Segment="6">
-        <DList Name="object_obj_yasi_DL_000360" Offset="0x360" />
-        <Texture Name="object_obj_yasi_TLUT_000630" OutName="tlut_000630" Format="rgba16" Width="4" Height="4" Offset="0x630" />
-        <Texture Name="object_obj_yasi_Tex_000650" OutName="tex_000650" Format="ci4" Width="32" Height="128" Offset="0x650" />
-        <Texture Name="object_obj_yasi_Tex_000E50" OutName="tex_000E50" Format="rgba16" Width="16" Height="16" Offset="0xE50" />
-        <Texture Name="object_obj_yasi_Tex_001050" OutName="tex_001050" Format="rgba16" Width="16" Height="16" Offset="0x1050" />
-        <Texture Name="object_obj_yasi_Tex_001250" OutName="tex_001250" Format="rgba16" Width="8" Height="8" Offset="0x1250" />
-        <Collision Name="object_obj_yasi_Colheader_001428" Offset="0x1428" />
+        <DList Name="gYasiDL" Offset="0x360" />
+        <Texture Name="gYasiTLUT" OutName="yasi_tlut" Format="rgba16" Width="4" Height="4" Offset="0x630" />
+        <Texture Name="gYasiFrondTex" OutName="yasi_frond_tex" Format="ci4" Width="32" Height="128" Offset="0x650" />
+        <Texture Name="gDekuNutTex" OutName="deku_nut_tex" Format="rgba16" Width="16" Height="16" Offset="0xE50" />
```

Comment:
```suggestion
        <Texture Name="gYasiDekuNutTex" OutName="yasi_deku_nut_tex" Format="rgba16" Width="16" Height="16" Offset="0xE50" />
```
Since this is not the only Deku Nut texture knocking around and could easily clash with something in gameplay_keep